src/site/views/event/tmpl/default.php

```diff
@@ -117,7 +117,10 @@
 							foreach ($eventAttendees as $person)
 							{
 								$universityCounts[$person['Uni']] += 1;
-								$levelCounts[$person['Level']] += 1;
+								if (strpos($person['Ticket'], 'Party') === false && strpos($person['Ticket'], 'party') === false)
```

Comment:
If you're set on doing it this way though. How about forcing the ticket name to lowercase rather than checking for "party" and "Party"? What if it was spelt "PArty"?
```suggestion
								if (strpos(strtolower($person['Ticket']), 'party') === false)
```
Disclaimer: just suggested in web browser; code not tested!

Reply:
yeah, duh, that's much cleaner (and it works, just tested)